README.md

```diff
@@ -194,6 +196,9 @@ expect (store.select ('root.10.year').get ()).to.equal (2016);
 expect (store.select ('root.12.name').get ()).to.equal ('bar');
 expect (store.select ('root.10').get ('value')).to.deep.equal ({year: 2016, name: 'foo'});
 expect (store.select ('root.13').get ('value')).to.equal ('none');
+
+// remove this collection with:
+store.applyCollection ('root', []);
```

Comment:
@SL you should provide a separate example for the special case where the value is set to `[]` and not remove the root node, but a child node instead. Add also an `expect ()` example of what the test shows.